.github/workflows/ci.yml

```diff
@@ -28,23 +28,17 @@ jobs:
         java: [ 8, 11 ]
         os: [ ubuntu-latest, macos-latest, windows-latest ]
         include:
-          - java: 12
-            os: ubuntu-latest
-          - java: 13
```

Comment:
why remoe this

Reply:
![image](https://github.com/apache/shenyu/assets/49482363/d7896f47-e7c4-4af9-841a-2dd8abece1c5)

Reply:
@yu199195 >_-
so, it's necessary or not?

Reply:
not remove eles jdk version